omnibus/config/software/aerospike-client.rb

```diff
@@ -9,16 +9,17 @@
   # The binary wheels on PyPI are not yet compatible with OpenSSL 1.1.0+, see:
   # https://github.com/aerospike/aerospike-client-python/issues/214#issuecomment-385451007
   # https://github.com/aerospike/aerospike-client-python/issues/227#issuecomment-423220411
-  command "git clone https://github.com/aerospike/aerospike-client-c.git #{install_dir}/embedded/lib/aerospike"
-  command "cd #{install_dir}/embedded/lib/aerospike && git checkout #{version}"
-  command "cd #{install_dir}/embedded/lib/aerospike && git submodule update --init"
+  command "git clone https://github.com/aerospike/aerospike-client-c.git ./aerospike"
```

Comment:
I vaguely recall, but I may be making this up, that omnibus supports git sources? Like I said, I'm getting old and this could be a fabrication of my mind.

Also, we could probably make this happen in a single command, and use a shallow clone (I'm not 100% sure the shallow-clone will work when specifying a branch, but it might be worth trying):

```
command "git clone -b #{version} --depth 1 https://github.com/aerospike/aerospike-client-c.git ./aerospike"
```

And do we also want to clone submodules shallowly? Anyway, just some ideas, feel free to ignore.

Reply:
There is indeed a git fetcher, let me try that.

Reply:
Did it work?

Reply:
Still waiting for my test pipeline before adding it to the PR's branch